src/AssetRegistry/StandardERC20PricingModule.sol

```diff
@@ -102,14 +106,23 @@ contract StandardERC20PricingModule is PricingModule {
      * @dev Since ERC20 tokens have no Id, the Id should be set to 0
      * @return A boolean, indicating if the asset passed as input is whitelisted
      */
-    function isWhiteListed(address asset, uint256) external view override returns (bool) {
-        if (isAssetAddressWhiteListed[asset]) {
+    function isWhiteListed(address asset, uint256) public view override returns (bool) {
+        if (isAssetAddressWhiteListed[asset].isWhiteListed) {
             return true;
         }
 
         return false;
     }
 
+    function processDeposit(address asset, uint256, uint256 amount) external onlyMainReg returns (bool success) {
```

Comment:
Same remarks for all pricing modules:
- Where is documentation?
- Where are the tests?